### PR TITLE
MGMT-10094: Update must gather images for the OLM operators

### PR DESCRIPTION
**Pull request by rewantsoni**
Update the must gather images for the OLM operators

Images are taken from
LSO: https://catalog.redhat.com/software/containers/openshift4/ose-local-storage-mustgather-rhel8/5ff374e34fb65bf4159e2c85

ODF: https://catalog.redhat.com/software/containers/odf4/ocs-must-gather-rhel8/614cda2c69cb9f1af5ba6ad3

CNV: https://catalog.redhat.com/software/containers/container-native-virtualization/cnv-must-gather-rhel8/5d491bbfbed8bd380993a4e0

Signed-off-by: Rewant Soni <resoni@redhat.com>

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @gamli75 
/cc @slaviered 

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] Reviewers have been listed
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [x] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: data/default_must_gather_versions.json [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 	"4.9": {
-		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v2.6.5",
-		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8",
-		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8"
+		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v4.9.3",
+		"odf": "registry.redhat.io/odf4/ocs-must-gather-rhel8:v4.9",
+		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8:v4.9"
+	},
+	"4.10": {
+		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v4.10.0",
```
Comment (pkliczewski) [inline data/default_must_gather_versions.json:18]: @tiraboschi please confirm whether the images are correct.

Comment (tiraboschi) [reply data/default_must_gather_versions.json:18]: +1

Comment (slaviered) [reply data/default_must_gather_versions.json:18]: ocs should be always present in the json. This is the default must gather image to use (not related to operators) and without  it the must-gather collection will not function propely

Also, why there are images missing in 4.10 If there are no new image release put there the versions from 4.9. We do not accumulate the versions so each entry should be a full description of the operator and ocs image inventory

**Review by slaviered — CHANGES_REQUESTED** [review]
see my comment in the thread

File: data/default_must_gather_versions.json [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
-		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8"
+		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8:v4.8",
+		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8:v4.8"
 	},
 	"4.9": {
-		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v2.6.5",
-		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8",
-		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8"
+		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v4.9.3",
+		"odf": "registry.redhat.io/odf4/ocs-must-gather-rhel8:v4.9",
```
Comment (gamli75) [inline data/default_must_gather_versions.json:14]: based on what @slaviered wrote - should it be ocs or odf?

File: data/default_must_gather_versions.json (hunk 1 above)
Comment (rewantsoni) [reply data/default_must_gather_versions.json:18]: I have updated the images based on the version, for 4.10 I have used must gather operator images for 4.10

File: data/default_must_gather_versions.json (hunk 2 above)
Comment (rewantsoni) [reply data/default_must_gather_versions.json:14]: based on what @slaviered wrote, it should be ocs. I have updated the PR

**Review by gamli75 — APPROVED** [review]

**Review by gamli75 — APPROVED** [review]

**Pull request merged**